supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,147 @@
+[[openshift-container-storage-conventions]]
+
+[discrete]
+[[backing-store]]
+==== backing store (noun)
+*Description*: A type of storage resource for Multicloud Object Gateway, which is used to store data, for example, from RADOS gateway (RGW), Amazon Web Services S3, Azure Blob Storage, IBM Cloud Object Storage. 
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[container-storage-interface]]
+==== Container Storage Interface (noun)
+*Description*: The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems like Kubernetes, and in particular Red Hat OpenShift Container Platform. This allows OpenShift Container Platform to consume storage from third-party storage providers that implement the CSI interface as persistent storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[multicloud-object-gateway]]
+==== Multicloud Object Gateway (noun)
+*Description*: Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform you can use to start with a small storage service and then scale according to your requirements on-premises, in multiple clusters, and with cloud-native storage.
+
+Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that you can use to start with a small storage service and then scale according to your requirements on-premise, in multiple clusters, and with cloud-native storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-bucket]]
+==== namespace bucket (noun)
+*Description*: Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data written to namespace buckets are kept plain and not deduped, compressed, or encrypted. The data can be consumed directly from the repository.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-store]]
+==== namespace store (noun)
+*Description*: A type of a resource for Multicloud Object Gateway that is used to store plain data and is used by namespace buckets. The supported stores are S3, Azure, and S3 compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket]]
+==== object bucket (noun)
+*Description*: A container for storing objects.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket-claim]]
+==== object bucket claim (noun)
+*Description*: Creates a new bucket and an application account in Multicloud Object Gateway or RADOS Gateway (RGW) with permissions to the bucket, including a new access key and secret access key. An object bucket claim can be used to request an S3 compatible bucket for the workloads.
```

Comment:
Passive > active suggestion:

`..You can use an object bucket claim to request an S3 compatible bucket for the workloads.`

or if it's some piece of the technology that is really doing the requesting, then something like this with the component defined:

`.. <X_component> uses an object bucket claim to request an S3 compatible bucket for the workloads.`

Reply:
Done. (it's the first option because users make use of CRs to use the object bucket claims to request S3 compatible buckets for the workloads.)